lib/lograge/log_subscribers/action_cable.rb

```diff
@@ -27,7 +25,7 @@ def default_status
       end
 
       def extract_runtimes(event, _payload)
-        { duration: event.duration.to_f.round(2) }
+        { duration: 1_000 * event.duration }
```

Comment:
Is this changing the `duration` to be in nanoseconds (ref: https://docs.datadoghq.com/logs/log_configuration/attributes_naming_convention/#performance)? If so, could you add a comment above it?

Reply:
Yessir. Will add a comment.